src/BrazeMessageComponent.tsx

```diff
@@ -15,8 +15,14 @@ import {
 } from './TheGuardianIn2020Banner';
 import { BrazeClickHandler } from './utils/tracking';
 
+import { COMPONENT_NAME as NEWSLETTER_EPIC_NAME, NewsletterEpic } from './NewsletterEpic';
```

Comment:
I wonder - does it make sense to have the `NewsLetterEpic` be exposed as a component that's usable by marketing at this point (same question about the Storybook story for this one). It feels like the final piece to having a truly generic component is to be able to use the image picker with it. What do you reckon?

Reply:
I've removed it from Storybook for this PR. In addition to the grid integration, we would probably want a user friendly way to select the newsletter id, so I think making it generic for marketing deserves it's own PR.